docs/content/concepts/resources.mdx

```diff
@@ -75,7 +75,12 @@ def asset_requires_resource(context):
 
 #### Providing resources to software-defined assets
 
-Resources can be provided to software-defined assets by passing them <PyObject object="Definitions" />. The resources provided to <PyObject object="Definitions" /> are automatically bound to the assets.
+How resources are provided to assets depends on how you're organizing your code definitions in Dagster.
```

Comment:
"code definitions" isn't a dagster noun but i think is fine here since it covers both cases